plugins/imagebase/plugin.js

```diff
@@ -586,16 +593,26 @@
 			 * @private
 			 * @member CKEDITOR.plugins.imagebase.featuresDefinitions.caption
 			 * @param {CKEDITOR.dom.element} sender The element that this function should be called on.
+			 * @param {Boolean} [isFocused] Indicates if current widget should be treated as a focused one.
```

Comment:
As it will land on different version than one in which `imagebase` plugin was introduced we should state here `since` which version this param is available.